src/Raven.Server/Config/RavenConfiguration.cs

```diff
@@ -317,7 +317,7 @@ IEnumerable<ConfigurationEntryMetadata> GetConfigurationEntryMetadataFor(Propert
                         foreach (var metadata in GetConfigurationEntryMetadataFor(configurationProperty, configurationPropertyType))
                             yield return metadata;
 
-                        yield break;
+                        continue;
```

Comment:
Can you elaborate more on this? What was the issue and how does this fix it?

Reply:
We have two nested configs inside `Monitoring` (`OpenTelemetry` and `Snmp`). With `yield break` we only scan the first nested config, which results in the OpenTelemetry config not being exposed to the studio.